docs/data/packages.json

```diff
@@ -1,12 +1,18 @@
 {
```

Comment:
@RishhiB These contents don't look right to me. We don't want our SDK customers knowing anything about our lower-level packages like `fluid-static` or `container-definitions`, or even our individual DDS packages - all of those contents are surfaced via the `fluid-framework` package. I believe the correct list is:

## V1
- fluid-framework
- @fluidframework/counter
- @fluidframework/azure-client
- @fluidframework/azure-service-utils
- @fluidframwork/tinylicious-client

## V2
- fluid-framework
- @fluidframework/azure-client
- @fluidframework/azure-service-utils
- @fluid-experimental/odsp-client